_drafts/zero-to-main-libc.md

```diff
@@ -0,0 +1,721 @@
+---
+title: "From zero to main(): Bootstrapping libc with Newlib"
+description:
+  "TODO"
+author: francois
+tags: [zero-to-main]
+---
+
+In this series of post, we’ve worked methodically to demystify what happens to
+firmware before the main() function is called. So far, we bootstrapped a C
+environment, wrote a linker script from scratch, and implemented our own
+bootloader.
```

Comment:
Tense nit, but if you are going to start with `we have worked`, you might want to change your later tense to `we have bootstrapped, written a linker script,`

Reply:
👍